<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>PDG_LocalisationCreator_GUIClass</class>
  <widget class="QMainWindow" name="PDG_LocalisationCreator_GUIClass">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>500</width>
        <height>250</height>
      </rect>
    </property>
    <property name="minimumSize">
      <size>
        <width>500</width>
        <height>250</height>
      </size>
    </property>
    <property name="maximumSize">
      <size>
        <width>500</width>
        <height>250</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>PDG Localisation Tool v2.1</string>
    </property>
    <property name="iconSize">
      <size>
        <width>64</width>
        <height>64</height>
      </size>
    </property>
    <widget class="QWidget" name="centralWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
          <widget class="QFrame" name="folderSelectionBox">
            <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
                <widget class="QLabel" name="outputPathLabel">
                  <property name="text">
                    <string>Output Folder:</string>
                  </property>
                </widget>
              </item>
              <item row="0" column="1">
                <widget class="QLineEdit" name="outputPathLineEdit"/>
              </item>
              <item row="0" column="2">
                <widget class="QPushButton" name="outputPathButton">
                  <property name="text">
                    <string>Browse...</string>
                  </property>
                </widget>
              </item>
              <item row="1" column="0">
                <widget class="QLabel" name="vanillaPathLabel">
                  <property name="text">
                    <string>Vanilla Files Folder:</string>
                  </property>
                </widget>
              </item>
              <item row="1" column="1">
                <widget class="QLineEdit" name="vanillaPathLineEdit"/>
              </item>
              <item row="1" column="2">
                <widget class="QPushButton" name="vanillaPathButton">
                  <property name="text">
                    <string>Browse...</string>
                  </property>
                </widget>
              </item>
            </layout>
          </widget>
        </item>
        <item>
          <widget class="QGroupBox" name="groupSheets">
            <property name="title">
              <string>Sheets</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_sheets">
              <item alignment="Qt::AlignHCenter">
                <widget class="QPushButton" name="selectSheetsButton">
                  <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="minimumSize">
                    <size>
                      <width>100</width>
                      <height>0</height>
                    </size>
                  </property>
                  <property name="text">
                    <string>Select Sheetsâ€¦</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLabel" name="sheetsSummaryLabel">
                  <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="text">
                    <string>No sheets selected</string>
                  </property>
                  <property name="alignment">
                    <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
                  </property>
                  <property name="wordWrap">
                    <bool>true</bool>
                  </property>
                </widget>
              </item>
            </layout>
          </widget>
        </item>
        <item>
          <widget class="QFrame" name="actionBox">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item alignment="Qt::AlignHCenter">
                <widget class="QPushButton" name="unifiedRunButton">
                  <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="minimumSize">
                    <size>
                      <width>200</width>
                      <height>44</height>
                    </size>
                  </property>
                  <property name="font">
                    <font>
                      <pointsize>12</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                    </font>
                  </property>
                  <property name="styleSheet">
                    <string notr="true">QPushButton#unifiedRunButton{
    padding: 10px 24px;
    color: white;
    background-color: rgb(24, 146, 0);
    border: 1px solid rgba(255,255,255,60);
    border-radius: 10px;
}
QPushButton#unifiedRunButton:hover{
    background-color: rgb(29, 168, 2);
}
QPushButton#unifiedRunButton:pressed{
    background-color: rgb(22, 128, 1);
}
QPushButton#unifiedRunButton:disabled{
    background-color: rgb(70,70,70);
    color: rgb(170,170,170);
    border: 1px solid rgba(255,255,255,30);
}</string>
                  </property>
                  <property name="text">
                    <string>ENGAGE</string>
                  </property>
                </widget>
              </item>
            </layout>
          </widget>
        </item>
      </layout>
    </widget>
  </widget>
  <resources/>
  <connections/>
</ui>